plugins/org.eclipse.papyrus.uml.interaction.model.edit/src-gen/org/eclipse/papyrus/uml/interaction/model/provider/MExecutionItemProvider.java

```diff
@@ -66,13 +66,13 @@ public MExecutionItemProvider(AdapterFactory adapterFactory) {
 	 * @generated
 	 */
 	protected void addStartPropertyDescriptor(Object object) {
-		itemPropertyDescriptors.add(createItemPropertyDescriptor(
```

Comment:
These formatting changes look like the "format code" property in "Templates and Merge" in the genmodel didn't work?  Which would be odd.  Maybe just explicitly format the `src-gen/` folder before committing?

Reply:
I actually tried formatting src-gen folder and it didn't lead to any changes -- and the version I committed is the result (with and without explicitly formatting the src-gen folders). Did we maybe commit the files without being explicitly formatted or without the format code property switched on initially?